components/podio/sources/common/common-poll.mjs

```diff
@@ -0,0 +1,63 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import app from "../../podio.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  props: {
+    app,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+    db: "$.service.db",
+  },
+  methods: {
+    getResourceFn() {
+      throw new Error("getResourceFn() is not implemented!");
+    },
+    getResourceFnArgs() {
+      throw new Error("getResourceFnArgs() is not implemented!");
+    },
+    doesResourceFnHavePaging() {
+      throw new Error("doesResourceFnHavePaging() is not implemented!");
+    },
+    getSummary() {
+      throw new Error("getSummary() is not implemented!");
+    },
+    getItemId() {
+      throw new Error("getItemId() is not implemented!");
+    },
+    _getLastID() {
+      return this.db.get("lastId") | 0;
+    },
```

Comment:
Is `|` supposed to be `||`?
```suggestion
    _getLastID() {
      return this.db.get("lastId") || 0;
    },
```

Reply:
Mostly I use `|` for this kind of things because it lets me have left hand side if it is `""`, `0` or other falsy value. But here it does not change anything so I will use `||` as you suggested